src/api-service/__app__/timer_tasks/__init__.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
```

Comment:
Is the shebang needed?

Reply:
This is common across the entire project.